server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/dto/InnaccessibleFieldException.java

```diff
@@ -0,0 +1,44 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.dto;
+
+import javax.mail.MessagingException;
+
+import org.apache.james.webadmin.dto.MailDto.AdditionalField;
+
+public class InnaccessibleFieldException extends Exception {
+
+    private final AdditionalField field;
+    private final MessagingException originalException;
+
+    public InnaccessibleFieldException(AdditionalField field, MessagingException originalException) {
```

Comment:
`originalException` sounds like `cause` to me.

It should be passed to the parent constructor to be part of the stacktrace, and generic.